hal/inc/hal_dynalib_ble.h

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2018 Particle Industries, Inc.  All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation, either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef HAL_DYNALIB_BLE_H
+#define	HAL_DYNALIB_BLE_H
+
+#include "dynalib.h"
+#include "hal_platform.h"
+
+#if HAL_PLATFORM_BLE
+
+#ifdef DYNALIB_EXPORT
+#include "ble_hal.h"
+#endif
+
+DYNALIB_BEGIN(hal_ble)
+
+DYNALIB_FN( 0,  hal_ble, ble_stack_init, int(void*) )
```

Comment:
Nitpicking: uncommon formatting. We don't add extra spaces before/after parenthesis.